tests/platform_tests/api/test_chassis_fans.py

```diff
@@ -10,6 +10,7 @@
 from tests.common.helpers.platform_api import chassis, fan
 
 from platform_api_test_base import PlatformApiTestBase
+from tests.platform_tests.thermal_control_test_helper import start_thermal_control_daemon, stop_thermal_control_daemon
```

Comment:
you could as well use "from thermal_control_test_helper import start_thermal_control_daemon, stop_thermal_control_daemon" as this is local.